code/modules/awaymissions/gateway.dm

```diff
@@ -233,6 +233,17 @@ GLOBAL_DATUM(the_gateway, /obj/machinery/gateway/centerstation)
 			M.client.move_delay = max(world.time + 5, M.client.move_delay)
 
 
+/obj/machinery/gateway/centeraway/admin

```

Comment:
This can just be removed, overriding the proc below makes this a subtype

Reply:
An empty declaration is better than going to the proc right away imo